### PR TITLE
Document how to add words to the Vale dictionary

### PR DESCRIPTION
**Pull request by jdbaldry**
Closes https://github.com/grafana/writers-toolkit/issues/669.

### CODE REVIEW

**Review by clayton-cornell — APPROVED** [review]

File: docs/sources/review/lint-prose/dictionary/add-words.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+      ```
+
+   1. Replace _`<AFFIXES>`_ with the concatenation of the Hunspell affixes.
+
+      To learn which affixes you can add, refer to the [Hunspell affixes table](https://grafana.com/docs/writers-toolkit/review/lint-prose/dictionary/#hunspell-affixes).
+
+      You must put the affixes between single quotes (`'`).
+
+      Some abbreviations have a plural suffix.
+      To add the plural suffix, include `s`.
```
Comment (clayton-cornell) [inline docs/sources/review/lint-prose/dictionary/add-words.md:198]: Is this case sensitive?

**Review by clayton-cornell — APPROVED** [review]
New content looks good.

Question: Should we suggest/recommend putting a description field with all words?

File: docs/sources/review/lint-prose/dictionary/add-words.md [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+# Add words to the Grafana Labs dictionary
+
+The Grafana Labs documentation team maintains a dictionary used for spell checking.
+The same dictionary is used to generate some Vale rules from the metadata in the word definition.
+
+The template file uses the Jsonnet programming language but you don't need to know Jsonnet to add a new word.
+
+This topic explains how to perform the following tasks:
+
+- [Add a verb](#add-a-verb)
```
Comment (GrafanaWriter) [inline docs/sources/review/lint-prose/dictionary/add-words.md:16]: Should this be 
```suggestion
- [Add a general word (noun, verb, adjective, or adverb) ](#add-a-verb)
```

File: docs/sources/review/lint-prose/dictionary/add-words.md [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+- [Add a product](#add-a-product)
```
Comment (GrafanaWriter) [inline docs/sources/review/lint-prose/dictionary/add-words.md:17]:
```suggestion
- [Add a product name](#add-a-product)
```

File: docs/sources/review/lint-prose/dictionary/add-words.md [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+
+- Clone the [Writers' Toolkit repository](https://github.com/grafana/writers-toolkit).
+
+  For more information, refer to [Create a local repository](https://grafana.com/docs/writers-toolkit/write/tooling-and-workflows/#create-a-local-repository).
+
+- Create a branch for your change.
+
+  For more information, refer to [Create a branch from the default remote branch](https://grafana.com/docs/writers-toolkit/write/tooling-and-workflows/#create-a-branch-from-the-default-remote-branch).
+
+## Add a verb
```
Comment (GrafanaWriter) [inline docs/sources/review/lint-prose/dictionary/add-words.md:33]:
```suggestion
## Add a general word (noun, verb, adjective, adverb)
```

File: docs/sources/review/lint-prose/dictionary/add-words.md [hunk 5]
```diff
@@ -0,0 +1,242 @@
...
+
```
Comment (GrafanaWriter) [inline docs/sources/review/lint-prose/dictionary/add-words.md:34]: Add words to the dictionary to ensure consistent and comprehensive coverage of words within our documentation. You can add a verb, noun, adverb, or adjective. To add product names or abbreviations, see those specific guidelines that follow.

File: docs/sources/review/lint-prose/dictionary/add-words.md [hunk 6]
```diff
@@ -0,0 +1,242 @@
...
+A verb generally describes an action.
+It may take prefixes and suffixes that you can add as part of the definition.
+
+### Steps
+
+To add a verb:
```
Comment (GrafanaWriter) [inline docs/sources/review/lint-prose/dictionary/add-words.md:40]:
```suggestion
To add a general word:
```

File: docs/sources/review/lint-prose/dictionary/add-words.md [hunk 7]
```diff
@@ -0,0 +1,242 @@
...
+
+      You must put the word stem between single quotes (`'`).
+
+      Your line should look similar to the following:
+
+      ```jsonnet
+      newWord('run', <AFFIXES>, <PART OF SPEECH>),
+      ```
+
+   1. Replace _`<AFFIXES>`_ with the concatenation of the Hunspell affixes.
```
Comment (GrafanaWriter) [inline docs/sources/review/lint-prose/dictionary/add-words.md:69]: Make this a note specific to verbs.

**Review by GrafanaWriter — COMMENTED** [review]
Expanded verb to more general words, and added a note section for the verb affixing of verbs (I think this the right spot for that note.

File: docs/sources/review/lint-prose/dictionary/add-words.md (hunk 1 above)
Comment (jdbaldry) [reply docs/sources/review/lint-prose/dictionary/add-words.md:201]: It is, I've added a note in f8f5cfdb512a8fe92da44025f924ce970f8257cc

**Pull request merged**